### PR TITLE
Adds Edit Chapter

### LINKED ISSUE #42: Create "Edit Chapter"
Same as the other ones! Also please update **sidebar.erb**.

### PR DESCRIPTION
**Pull request by brunofbrito**
Fixes #42

### CODE REVIEW

**Comment by ricardojusto** [comment]
image

**Comment by rumama** [comment]
:+1:

File: app/views/backend/sidebar.erb [hunk 1]
```diff
@@ -7,9 +7,11 @@
 				<ul class="list-unstyled">
 					<% all_courses.each do |course| %>
 					<li><a href="/course/show/<%= course.id %>"><%= course.name %></a></li>
-					<li><a href="/chapter/add/<%= course.id %>"><span class="glyphicon glyphicon-plus"></span> Add Chapter</li>
 					<li><a href="/course/edit/<%= course.id %>"><span class="glyphicon glyphicon-pencil"></span> Edit Course</a></li>
 					<br>
```
Comment (rumama) [inline app/views/backend/sidebar.erb]: unneeded break line element! If space is needed between elements, add a class to the element and add margin/padding.

**Comment by brunofbrito** [comment]
Fixed!

**Pull request merged**